packages/sitecore-jss/src/layout/rest-layout-service.ts

```diff
@@ -6,70 +6,6 @@ import { AxiosDataFetcher, AxiosDataFetcherConfig } from '../axios-fetcher';
 import { HttpDataFetcher, fetchData } from '../data-fetcher';
 import debug from '../debug';
 
-/**
- * Resolves layout service url
- * @param {LayoutServiceConfig} [options] layout service options
- * @param {string} apiType which layout service API to call ('render' or 'placeholder')
- * @returns the layout service url
- */
-export function resolveLayoutServiceUrl(
-  options: LayoutServiceConfig = {},
-  apiType: 'render' | 'placeholder'
-): string {
-  const { host = '', configurationName = 'jss', serviceUrl } = options;
-
-  if (serviceUrl) {
-    return serviceUrl;
-  }
-
-  return `${host}/sitecore/api/layout/${apiType}/${configurationName}`;
-}
-
-/**
- * Makes a request to Sitecore Layout Service for the specified route item path.
- * @deprecated Will be removed in a future release. Please use LayoutService.fetchLayoutData instead,
- * @see {LayoutService} - fetchLayoutData
- * @param {string} itemPath
- * @param {LayoutServiceRequestOptions<LayoutServiceData>} options
- * @returns {Promise<LayoutServiceData>} layout data
- */
-export function fetchRouteData(
-  itemPath: string,
-  options: LayoutServiceRequestOptions<LayoutServiceData>
-): Promise<LayoutServiceData> {
-  const { querystringParams, layoutServiceConfig } = options;
-
-  const fetchUrl = resolveLayoutServiceUrl(layoutServiceConfig, 'render');
-
-  return fetchData(fetchUrl, options.fetcher, { item: itemPath, ...querystringParams });
-}
-
-/**
- * Makes a request to Sitecore Layout Service for the specified placeholder in
- * a specific route item. Allows you to retrieve rendered data for individual placeholders instead of entire routes.
- * @deprecated Will be removed in a future release. Please use LayoutService.fetchPlaceholderData instead,
- * @see {LayoutService} - fetchPlaceholderData
- * @param {string} placeholderName
- * @param {string} itemPath
- * @param {LayoutServiceRequestOptions<PlaceholderData>} options
- * @returns {Promise<PlaceholderData>} placeholder data
- */
-export function fetchPlaceholderData(
-  placeholderName: string,
-  itemPath: string,
-  options: LayoutServiceRequestOptions<PlaceholderData>
-): Promise<PlaceholderData> {
-  const { querystringParams, layoutServiceConfig } = options;
-
-  const fetchUrl = resolveLayoutServiceUrl(layoutServiceConfig, 'placeholder');
-
-  return fetchData(fetchUrl, options.fetcher, {
-    placeholderName,
-    item: itemPath,
-    ...querystringParams,
-  });
-}
-
 export interface LayoutServiceConfig {
```

Comment:
Just noticed the other interface `LayoutServiceRequestOptions` in this file can be deleted. Also, probably the `LayoutServiceConfig`, `FetchParams`, and `FetchOptions` are artifacts of the old dataApi and can be simplified. Really the only one that's useful is probably the `FetchParams`. The rest seem redundant / unnecessary now.

Reply:
Good call, I removed redundant interfaces